daemon/container.go

```diff
@@ -649,12 +637,11 @@ func (container *Container) ReleaseNetwork() {
 	if container.Config.NetworkDisabled || !container.hostConfig.NetworkMode.IsPrivate() {
 		return
 	}
-	eng := container.daemon.eng
 
-	job := eng.Job("release_interface", container.ID)
-	job.SetenvBool("overrideShutdown", true)
-	job.Run()
-	container.NetworkSettings = &NetworkSettings{}
+	_ = bridge.Release(container.ID)
```

Comment:
it was to make it obvious the error was ignored. I can remove it if you don't want it.